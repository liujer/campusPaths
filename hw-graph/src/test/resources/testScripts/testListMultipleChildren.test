# Tests the method listChildren of DLGraph by adding a
# node with multiple children
CreateGraph A
AddNode A one
AddNode A two
AddNode A three
AddEdge A one two e12
AddEdge A one three e13
ListChildren A one
ListChildren A two
ListChildren A three
