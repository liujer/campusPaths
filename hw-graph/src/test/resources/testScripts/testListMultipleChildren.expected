# Tests the method listChildren of DLGraph by adding a
# node with multiple children
created graph A
added node one to A
added node two to A
added node three to A
added edge e12 from one to two in A
added edge e13 from one to three in A
the children of one in A are: three(e13) two(e12)
the children of two in A are:
the children of three in A are: